### DISCUSSION TITLE
Error of files generated by pywinrt.exe

### DISCUSSION THREAD

**Discussion by sotanakamura** [pywinrt/pywinrt#21] [Q&A] [answered]
@dlech 
I generated files using pywinrt.exe

```
pywinrt.exe -input sdk
```

I runned the following sample code.

```py
import sys
sys.path.append("output")
import winrt.windows.foundation as wf
u = wf.Uri("https://github.com/")
u2 = u.combine_uri("Microsoft/xlang/tree/master/src/tool/python")
print(str(u2))
```

I got the error.

```
Traceback (most recent call last):
  File "C:\Users\Sota\git\winui-python\uri.py", line 3, in <module>
    import winrt.windows.foundation as wf
  File "C:\Users\Sota\git\winui-python\output\winrt\windows\foundation\__init__.py", line 5, in <module>
    import winrt.system
  File "C:\Users\Sota\git\winui-python\output\winrt\system\__init__.py", line 9, in <module>
    from .._winrt import __file__ as _winrt_file, Array as Array, Object as Object
ModuleNotFoundError: No module named 'winrt._winrt'
```

How can I solve this error? How can I use the files generated by pywinrt.exe?

**Comment by dlech** [comment, marked as answer]
Did you compile the generated code? Did you copy the .pyd file from the build directory to the proper location? Are you using the matching Python runtime compared to the one used to compile, e.g. Debug vs. Release?

**Reply by sotanakamura** [reply]
Thank you for your reply. To be honest, I don't know python well and I thought I can use it by just importing like cppwinrt.

**Reply by dlech** [reply]
If you just want to use the APIs, install the `winsdk` package from PyPi. https://pypi.org/project/winsdk/
